daphne/src/vdaf/mod.rs

```diff
@@ -127,6 +137,15 @@ impl AsRef<[u8]> for VdafVerifyKey {
     }
 }
 
+impl AsMut<[u8]> for VdafVerifyKey {
+    fn as_mut(&mut self) -> &mut [u8] {
+        match self {
+            Self::Prio3(ref mut bytes) => &mut bytes[..],
+            Self::Prio2(ref mut bytes) => &mut bytes[..],
```

Comment:
are these `ref mut` necessary? I think you can omit them

Reply:
YOu're right, removed.